tnth/prodanon/tests/test_users.py

```diff
@@ -0,0 +1,14 @@
+from prodanon.transformers.users import email_for_user
```

Comment:
Very appreciative of the tests and docker automation!

I'd be happy to add CI/CD but I think it'd make more sense as a separate repo at that point, and wasn't sure if you were waiting for this utility to become more generalized/mature first

Reply:
add if you'd like.  i needed these tests during development.  probably best to have CI running them one way or the other.